glean-core/csharp/Glean/LibGleanFFI.cs

```diff
@@ -161,6 +161,47 @@ bool disabled
              string storage_name
          );
 
+        // Custom pings
+
+        /// <summary>
+        /// This implements the developer facing API for custom pings.
```

Comment:
This comment is wrong: this may refer to the `PingMetricType` class itself, but this is just the handle.

Reply:
I hope it makes sense now.